2017/Cambridge-OMERO-API/index.html

```diff
@@ -0,0 +1,268 @@
+                        <li>Jupyter Notebooks
+                            <ul><li>Python/CellProfiler | R</li></ul>
+                        </li>
+                    </ol>
+                </section>
+
+
+                <section>
+                    <h2>OMERO is a server with clients</h2>
+
+                    <img src="https://downloads.openmicroscopy.org/presentations/2014/technique_tapas_nov_5/images/omero-platform-simple.png">
+                </section>
+
+
+                <section data-background="https://downloads.openmicroscopy.org/presentations/2014/Technique-Tapas-OMERO-Figure/images/webclient3.png">
+                </section>
+
+
+                <section>
+                    <h2>OMERO model objects</h2>
+                    <img src="images/developer-model-pdi.png">
+                    <br>
+                    See <a href="http://docs.openmicroscopy.org/omero/5.4.0/developers/Model/EveryObject.html">
+                        All Objects
+                    </a>
+                </section>
+
+
+
+                <section>
+                    <h2>Remote Objects & Proxy Services</h2>
+                    <img src="images/omero-proxy-objects.png"><br>
+                    <a href="http://docs.openmicroscopy.org/latest/omero/developers/index.html">
+                        Developer Documentation
+                    </a>
+                </section>
+
+
+                <section>
+                    <h2>Connecting to OMERO: Python setup</h2>
+                    <ul>
+                        <li>Download
+                            <a href="https://downloads.openmicroscopy.org/latest/omero5/py.zip">
+                                OMERO.py libs
+                            </a>, add to PYTHONPATH
+                        </li>
+                        <li>
+                            <code class="python">$ pip install zeroc-ice&gt;3.5,&lt;3.7</code>
+                        </li>
+                    </ul>
+                </section>
+
+
+                <section>
+                    <h2>Connecting to OMERO: Python code</h2>
+                    <pre><code class="python">import omero
+
+client = omero.client("localhost")
+# login to create a session (Service Factory)
+session = client.createSession("username", "password")
+# get a Stateless Service
+queryService = session.getQueryService()
+# get a local omero.model.DatasetI
+dataset = queryService.get("Dataset", 1)
+print dataset               # see what's loaded
+print dataset.name.val      # rtypes same as getName().getValue()
+print dataset.copyImageLinks()      # omero.UnloadedEntityException!
+
+client.closeSession()           # Free server resourses
+</code></pre>
+                </section>
+
+
+                <section>
+                    <h2>Connecting to OMERO: Java Setup</h2>
+                    <ul>
+                        <li>Download
+                            <a href="https://downloads.openmicroscopy.org/latest/omero/artifacts">
```

Comment:
This requires a bit of clarification
Since basically the omero_client.jar that is need is in the Matlab toolbox

Reply:
A good example https://github.com/mtbc/omero-downloader